documentation/blog/2025-08-10-vibe-coding-with-goose-building-apps-with-ai-agents/index.md

```diff
@@ -0,0 +1,424 @@
+---
+title: How 7 AI Agents Worked Together to Build an App in One Hour
+description: "Learn how to build a full-stack AI app in under an hour using Goose's sub-agent orchestration, from planning to testing."
+authors: 
+    - angie
+---
+
+![blog cover](header-image.png)
+
+What if I told you that you could build a complete, working web application in under an hour using nothing but AI agents? Not just a simple "Hello World" app, but a full-stack application with a backend API, responsive frontend, unit tests, and documentation?
+
+That's exactly what we accomplished during the Vibe Coding workshop at [Berkeley's Agentic AI Summit](https://www.youtube.com/embed/_w5m3h9jY-w?start=7146), where I demonstrated how to use Goose's sub-agent orchestration to spin up an entire development team of AI agents. Each agent took on a specific role - from product planning to QA testing - and worked together to build "AI BriefMe", a web app that generates executive style briefings on any topic.
+
+<!-- truncate -->
+
+## The Power of Multi-Agent Development
+
+Traditional AI coding assistants are great at helping you write individual functions or debug specific issues. But what if you need to build something from scratch? What if you want to simulate an entire software development lifecycle?
+
+That's where Goose's sub-agent feature shines. Instead of doing everything yourself, you can orchestrate a team of specialized AI agents, each with their own expertise:
```

Comment:
```suggestion
That's where Goose's subagent feature shines. Instead of doing everything yourself, you can orchestrate a team of specialized AI agents, each with their own expertise:
```